File: SPSurvey.Core/SDK/SurveyItem.vb
Imports ControlOrigins.COUtility

Public Class SurveyItem

    Public Property SurveyID As Integer
    Public Property SurveyNM As String
    Public Property SurveyDS As String
    Public Property SurveyShortNM As String
    Public Property CompletionMessage As String
    Public Property AutoAssignFilter As String
    Public Property ResponseNMTemplate As String
    Public Property ReviewerAccountNM As String
    Public Property UseSurveyGroupsFL As Boolean
    Public Property EndDT As Nullable(Of Date)
    Public Property StartDT As Nullable(Of Date)
    Public Property ModifiedID As Integer
    Public Property ParentSurveyID As Integer?
    Public Property QuestionGroupList As New List(Of QuestionGroupItem)
    Public Property QuestionList As New List(Of QuestionItem)
    Public Property EmailTemplateList As New List(Of SurveyEmailTemplateItem)
    Public Property ReviewStatusList As New List(Of SurveyReviewStatusItem)
    Public Property StatusList As New List(Of SurveyStatusItem)
    Public Property SurveyType As New SurveyTypeItem
    Public Property ApplicationLookup As New List(Of LookupItem)
    Public Property ApplicationCount As Integer
    Public Property SurveyResponseCount As Integer
    Public Property QuestionCount As Integer
    Public Property QuestionGroupCount As Integer


End Class

File: ControlOrigins.Survey.Common/SDK/SurveyItem.cs
using System;
using System.Collections.Generic;

namespace ControlOrigins.Survey.Common.SDK
{
    public class SurveyItem
    {
        public int SurveyID { get; set; }
        public string SurveyNM { get; set; }
        public string SurveyDS { get; set; }
        public string SurveyShortNM { get; set; }
        public string CompletionMessage { get; set; }
        public string AutoAssignFilter { get; set; }
        public string ResponseNMTemplate { get; set; }
        public string ReviewerAccountNM { get; set; }
        public bool UseSurveyGroupsFL { get; set; }
        public DateTime? EndDT { get; set; }
        public DateTime? StartDT { get; set; }
        public int ModifiedID { get; set; }
        public int? ParentSurveyID { get; set; }
        public List<QuestionGroupItem> QuestionGroupList { get; set; } = new List<QuestionGroupItem>();
        public List<QuestionItem> QuestionList { get; set; } = new List<QuestionItem>();
        public List<SurveyEmailTemplateItem> EmailTemplateList { get; set; } = new List<SurveyEmailTemplateItem>();
        public List<SurveyReviewStatusItem> ReviewStatusList { get; set; } = new List<SurveyReviewStatusItem>();
        public List<SurveyStatusItem> StatusList { get; set; } = new List<SurveyStatusItem>();
        public SurveyTypeItem SurveyType { get; set; } = new SurveyTypeItem();
        public int ApplicationCount { get; set; }
        public int SurveyResponseCount { get; set; }
        public int QuestionCount { get; set; }
        public int QuestionGroupCount { get; set; }
    }
}